### ISSUE TITLE
Register the confirmation mail template 

### ISSUE THREAD

**Issue by alxy** [gergo85/oc-news#69]
Is there any reason why you don't register the confirmation mail template as well: https://github.com/gergo85/oc-news/blob/master/Plugin.php#L144-L145 ?

**Comment by alxy** [comment]
Yeah, i get that. But what I propose to do is register the cofirmation templates as well ( https://github.com/gergo85/oc-news/blob/master/views/mail/confirmation_en.htm ). Otherwise we cannot adjust the content and the appearance of this cofnrimation mail, and I surely want to customize it. Especially if I need it to be in a language that is neither english nor hungarian ;)

**Comment by gergo85** [comment]
I agree with you. It is an old solution, but I haven't got enough time to find better solution.
